packages/css/src/components/blockquote/README.md

```diff
@@ -1,14 +1,14 @@
 # Blockquote
 
-De blockquote (in het Nederlands: citaat) wordt gebruikt om uitspraken te accentueren. Een citaat is anders vormgegeven dan de broodtekst, zodat deze opvalt.
+We use blockquotes to emphasize citations.
+A quote appears differently from the main text to make it stand out.
 
-Let op: dit component voegt automatisch aanhalingstekens aan de tekst toe, dat hoef je niet zelf te doen.
+## Guidelines
 
-## Richtlijnen
+- Quote someone’s literal written or oral text.
+- A quote has a width of half or a whole column of text.
+- The component adds the correct quotation marks.
 
-- Bij een citaat haal je letterlijk een schriftelijke of mondelinge tekst van iemand aan.
-- Een citaat heeft een breedte van een halve of hele tekstkolom.
+## Relevant WCAG requirements
 
-## Relevante WCAG eisen
-
-- [WCAG 3.1.2](https://www.w3.org/WAI/WCAG21/Understanding/language-of-parts.html): wanneer het citaat in een andere taal is geschreven dan de pagina, dan moet het `lang` attribuut gebruikt worden om de taal duidelijk te maken.
+- [WCAG 3.1.2](https://www.w3.org/WAI/WCAG21/Understanding/language-of-parts.html): when the quote is written in a different language than the page, it should be `lang` attribute can be used to clarify the language.
```

Comment:
it should be 'lang' is a weird translation. Iets als "it should have a 'lang' attribute?